### ISSUE TITLE
Downed state ideal!

### ISSUE THREAD

**Issue by GothiccBih** [New-Chance-RP/Feedback#737]
<h2>Feature you are requesting.</h2>

Have it so when EMS there are at least 2 or more ems clocked on to get rid of the carry mechanic if you cry out for help. 

<h2>What are the benefits to your feature?</h2>

Speaking on both sides of the spectrum, being EMS and a civ that it would be best for everyone that if there are two or more EMTs clocked on, that the option for carrying gets canceled if you call for EMS. My reasons behind this are that there might just be one EMT clocked on dealing with a situation where there are several cops down or even civilians down and unable to respond to other calls. There is no way for the patient to know if you're responding to a call or not let alone knowing if the situation is even remotely close to being done this will make it a bit fairer to civilians and EMTs so that there can still be good medical RP going on!

<h2>Anything to support your feature request?</h2>

I like the new updates to EMS but believe this maybe good too! Thank you for reading.

**Comment by Senlarus** [comment]
Maybe it's because I'm its late and it's been a long day for me but I'm not sure I understand.

So if there are 2 or more ETM's on you think we should completely remove the option to carry people?

**Comment by Senlarus** [comment]
Or are you saying that with only 1 EMT on the carry option isn't cancelled no matter what? If that's the case, we felt it was more important to favor EMS in this situation. Basically... if you are going to call for help you are making the choice to wait for EMS. We left it so you could choose not to call for them...

But if we let you call for them AND also let you still get carried that'll be bad for EMS at some point and that's not what we want to support. **NEVER** should EMS show up to a call for help and have no one there because they were carried by a friend back to the hospital. With other people possibly wanting medical attention too, the last thing we want is that solo med on duty constantly responding to calls for EMS where the patient is magically gone when they arrive.

So basically, if you wanna call for help then you need to wait for it to arrive or hold E to take the loss at then end of the death timer. You're protected from EMS logging off because once they do you can carry again.

**Comment by GothiccBih** [comment]
> Maybe it's because I'm its late and it's been a long day for me but I'm not sure I understand.
> 
> So if there are 2 or more ETM's on you think we should completely remove the option to carry people?

Sorry, it's because I worded it poorly! I don't want the carry feature to be gone. What I meant was like if there is one and only one EMT clocked on and they are busy wrapped up in a situation where they may be busy for longer than 10 minutes like with a police situation for example where the EMT is on standby waiting for things to be cleared for them to arrive, once some in a completely different situation doesn't know and won't be able to receive help due to the fact of them calling out for EMS help while they are in a situation.

**Comment by GothiccBih** [comment]
I'm sorry if I am wording this all poorly trying to find a better way of explaining!

https://user-images.githubusercontent.com/66048097/103621563-02765780-4efb-11eb-82c6-61052fb79c89.mp4

**Comment by Senlarus** [comment]
I think I see what you mean. This current system was decided on by the leadership group with a request from EMS so we will have to discuss these changes again. I think I have something that can accomplish what you are concerned about but also ensure EMS isn't going to end up with an empty 10-52. If we can accommodate both concerns with a meet-in-the-middle solution that'd be the best.
